### DISCUSSION TITLE
Set (change) repeat after declaration

### DISCUSSION THREAD

**Discussion by evanderkuy** [sstaub/TickTwo#14] [Ideas]
Can I change the repeat parameter after declaration? Just like it is possible to change the interval.

Thanks for your nice library, changed all my timer functions to yours, but one I can not change because of the above needed functionality.

Etienne

**Comment by sstaub** [comment]
Not in the moment, I will make an issue.

**Comment by evanderkuy** [comment]
Ok, thank you!

**Reply by evanderkuy** [reply]
Can you also add a callback onStopped function when the timer has stopped/ended?

**Comment by sstaub** [comment]
I will have a look into this.
